.vscode/launch.json

```diff
@@ -27,7 +27,8 @@
             ],
             "port": 3001,
             "host": "localhost",
-            "justMyCode": false
+            "justMyCode": false,
+            "subProcess": true
```

Comment:
What is the purpose of the `subProccess`?

Reply:
I saw [this issue initially](https://github.com/microsoft/ptvsd/issues/1890#issuecomment-548176248) about Debugger exiting and this being a possible fix for it. It probably isn't necessary but it also doesn't seem to hurt.

Reply:
I can remove it as I don't think it makes a difference. I also noticed [ptvsd is deprecated](https://github.com/microsoft/ptvsd) as of 7 months ago and [debugpy](https://github.com/microsoft/debugpy/) replaces it. Will create a new issue for that.